.gitignore

```diff
@@ -1,4 +1,5 @@
 *.parser-cache
 www.pid
 vendor/
-.DS_Store
\ No newline at end of file
+.DS_Store
+composer.lock
```

Comment:
Just confirming, but I assume this is because of this advice? https://getcomposer.org/doc/02-libraries.md#lock-file

Reply:
The lock file was locking in old versions of dependencies, I think it's better to ignore it and only have composer.json express desired versions, and let composer choose the latest compatible version for the given HHVM version. This is also important to allow testing on multiple versions (I'll want to add the latest LTS, 4.102, to the list soon).